aomi/seed.py

```diff
@@ -92,7 +92,7 @@ def delete(client, path, opt):
         if vault_exception.errors[0] == 'permission denied':
             error_output("Permission denied deleting %s" % path, opt)
         else:
-            raise vault_exception
+            raise
```

Comment:
That was changed in response to `raising-bad-type` pylint issues in https://travis-ci.org/Autodesk/aomi/jobs/222150480

In this case it's raising the caught exception.